docs/dashboard.md

```diff
@@ -0,0 +1,7 @@
+# Dashboard Documentation
+
+### Format & Server Overview
+
+The main source of communication will come from a Flask server that contains two endpoints: `data` and `dashboard`. The server will serve JSON.
```

Comment:
Please provide specification for these endpoints.  I assume neither has parameters, but what are the return types?  

I'm guessing that `dashboard` is returning HTML - if so, no need to go into detail.

Reply:
@jonahb13 - when you have a chance, can you read this over and make sure I described the data endpoint correctly? Wasn't sure if the "200" in the return statement of your function was anything particular, or just a number you were giving it temporarily...